c/value.c

```diff
@@ -95,6 +95,8 @@ bool valuesEqual(Value a, Value b) {
       return AS_OBJ(a) == AS_OBJ(b);
 //< Hash Tables equal
   }
+
+  return false;
```

Comment:
I think this `return false;` needs to be outside the `#if` block in order to be included in chapter 18.